va/va_test.go

```diff
@@ -844,20 +844,30 @@ func setup() (*ValidationAuthorityImpl, *mocks.Statter, *blog.Mock) {
 	return va, stats, logger
 }
 
-func TestGetCAASetFallback(t *testing.T) {
+func TestCheckCAAFallback(t *testing.T) {
 	testSrv := httptest.NewServer(http.HandlerFunc(mocks.GPDNSHandler))
 	defer testSrv.Close()
 
+	stats := mocks.NewStatter()
+	logger := blog.NewMock()
 	caaDR, err := cdr.New(metrics.NewNoopScope(), time.Second, 1, nil, blog.NewMock())
 	test.AssertNotError(t, err, "Failed to create CAADistributedResolver")
 	caaDR.URI = testSrv.URL
 	caaDR.Clients["1.1.1.1"] = new(http.Client)
-	va, _, _ := setup()
-	va.caaDR = caaDR
+	va := NewValidationAuthorityImpl(
+		&cmd.PortConfig{},
+		nil,
+		nil,
+		caaDR,
```

Comment:
It looks like this tests the "CAA service is enabled" case. Can you add a test for the "CAA service is not enabled" case?